strings/base_collections_map.h

```diff
@@ -43,6 +43,18 @@ namespace winrt::impl
 
 WINRT_EXPORT namespace winrt
 {
+    template <typename K, typename V, typename Compare = std::less<K>, typename Allocator = std::allocator<std::pair<K const, V>>>
+    Windows::Foundation::Collections::IMap<K, V> map_view()
```

Comment:
When the function's name is `map_view`, I'd expect it to return `IMapView`, not `IMap`.

Reply:
Oops. Will fix.